packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -644,6 +644,22 @@
             "name": "aws_hideRegion",
             "description": "Records a call to remove a region from the explorer"
         },
+        {
+            "name": "serverlessland_createProject",
+            "description": "Called when creating a new Serverless Project"
+        },
+        {
+            "name": "serverlessland_downloadPattern",
+            "description": "Called when downloading a Serverless Pattern"
+        },
+        {
+            "name": "serverlessland_readme",
+            "description": "Called when opening the Serverless Pattern Readme"
+        },
+        {
+            "name": "serverlessland_wizard",
```

Comment:
is there an existing metric that could be used? check here: https://github.com/aws/aws-toolkit-common/blob/main/telemetry/definitions/commonDefinitions.json

Reply:
In the current metrics implementation, there are separate telemetry data streams to record the usage and behavior of different wizard features. So I have kept this to record telemetry for Serverless Land Wizard